modules/gdscript/gdscript_parser.cpp

```diff
@@ -527,6 +527,8 @@ GDScriptParser::Node *GDScriptParser::_parse_expression(Node *p_parent, bool p_s
 						return nullptr;
 					} else if (ScriptCodeCompletionCache::get_singleton()) {
 						res = ScriptCodeCompletionCache::get_singleton()->get_cached_resource(path);
+					} else {
+						res = ResourceLoader::load(path);
```

Comment:
I'm not familiar with this code but I feel like the comment in the `!validating` branch:
```
//this can be too slow for just validating code
```
actually explains why this `else` branch does not do a `load`, which is expensive.

CC @vnen @adamscott (note: 3.x code)

Reply:
Same thought, but better then failing. Since this only affects autoloaded Nodes that uses `preload`, it seems okay to me